### DISCUSSION TITLE
"Moving Day" Command for NPCs

### DISCUSSION THREAD

**Discussion by gerakis** [FakeFishGames/Barotrauma#7615] [Ideas and suggestions]
I want a command that makes every NPC pick up any crates that are on the ground and just hold them for when its time to change subs. Then when you load the new sub you have all your boxes which the NPCs just drop. That way you can move a bunch of crates at once even in singleplayer.

Some kind of item transfer service like how the store can pull items from anywhere on the sub would be good too.

**Comment by mLuby** [comment]
That Order exists, it's the ["Follow Me"](https://barotraumagame.com/wiki/Orders#Follow) Order. I've used it for exactly the scenario you're describing.
![follow order icon](https://barotraumagame.com/baro-wiki/images/thumb/c/c5/Orders_follow.png/64px-Orders_follow.png)
